gusto/linear_solvers.py

```diff
@@ -442,6 +442,11 @@ class IncompressibleSolver(TimesteppingSolver):
     @timed_function("Gusto:SolverSetup")
     def _setup_solver(self):
         equation = self.equations      # just cutting down line length a bit
+
+        # flag to indicate whether we're solving the compressible
```

Comment:
Very minor point, but since you only use `compressible` in one place below, it's probably not worth creating a new variable?

Reply:
I love minor points - done!